app/views/layouts/application.html.erb

```diff
@@ -4,12 +4,54 @@
     <title>MembersOnly</title>
     <%= csrf_meta_tags %>
     <%= csp_meta_tag %>
-
+    <%= stylesheet_link_tag "https://maxcdn.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css" %>
     <%= stylesheet_link_tag 'application', media: 'all', 'data-turbolinks-track': 'reload' %>
     <%= javascript_pack_tag 'application', 'data-turbolinks-track': 'reload' %>
   </head>
 
   <body class="bg-dark"> 
+      <% if flash[:notice] %>
+        <div class="notification is-primary global-notification text-light text-center">
+          <p class="notice"><%= notice %></p>
+        </div>
+
+      <% end %>
+      <% if flash[:alert] %>
+        <div class="notification is-danger global-notification text-danger text-center">
+          <p class='alert'><%= alert %></p>
+        </div>
```

Comment:
Please move logic into view helpers

Reply:
?????

Reply:
There is a `helpers` directory right after the `controllers`. Kindly implement the comments

Reply:
Done.